test/e2e/acl_logging.go

```diff
@@ -126,14 +121,59 @@ var _ = Describe("ACL Logging for NetworkPolicy", func() {
 			clientPodScheduledPodName := pods[pokerPodIndex].Spec.NodeName
 			composedPolicyNameRegex := fmt.Sprintf("%s_%s", nsName, egressDefaultDenySuffix)
 			Eventually(func() (bool, error) {
-				return assertAclLogs(
+				return assertACLLogs(
 					clientPodScheduledPodName,
 					composedPolicyNameRegex,
 					denyACLVerdict,
 					updatedAllowACLLogSeverity)
 			}, maxPokeRetries*pokeInterval, pokeInterval).Should(BeTrue())
 		})
 	})
+
+	When("the namespace's ACL logging annotation is removed", func() {
+		BeforeEach(func() {
+			By("removing the ACL logging annotation")
+			Expect(setNamespaceACLLogSeverity(fr, nsName, "", "", aclRemoveOptionDelete)).To(Succeed())
+		})
+
+		It("the ACL logs are updated accordingly", func() {
+			clientPod := pods[pokerPodIndex]
+			pokedPod := pods[pokedPodIndex]
+			clientPodScheduledPodName := pods[pokerPodIndex].Spec.NodeName
+			composedPolicyNameRegex := fmt.Sprintf("%s_%s", nsName, egressDefaultDenySuffix)
+
+			By("Counting the current number of log entries that match")
+			startCount, err := countACLLogs(
+				clientPodScheduledPodName,
+				composedPolicyNameRegex,
+				denyACLVerdict,
+				"")
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Poking some more from the client pod to the destination pod")
+			framework.Logf(
+				"Poke pod %s (on node %s) from pod %s (on node %s)",
+				pokedPod.GetName(),
+				pokedPod.Spec.NodeName,
+				clientPod.GetName(),
+				clientPod.Spec.NodeName)
+			Expect(
+				pokePod(fr, clientPod.GetName(), pokedPod.Status.PodIP)).To(HaveOccurred(),
+				"traffic should be blocked since we only use a deny all traffic policy")
+
+			Consistently(func() (bool, error) {
+				newCount, err := countACLLogs(
+					clientPodScheduledPodName,
+					composedPolicyNameRegex,
+					denyACLVerdict,
+					"")
+				if err != nil {
+					return false, err
+				}
+				return startCount == newCount, nil
+			}, maxPokeRetries*pokeInterval, pokeInterval).Should(BeTrue())
```

Comment:
Stood up to me that we could not use `isCountUpdatedAfterPokeExternalHost` or similar function here. Non-binding comment, just a remark.

Reply:
done

Reply:
done